src/components/UiBreakpoints/Theme/media.ts

```
@@ -0,0 +1,21 @@
+  xl: `@media (min-width: ${bp.xl}px)`,
+  custom: {
+    max968: '@media (max-width: 968px)',
+    max1130: '@media (max-width: 1130.98px)',
+    min426: '@media (min-width: 426px)',
+    min468: '@media (min-width: 468px)',
+    min641: '@media (min-width: 641px)',
+    min969: '@media (min-width: 969px)',
+    min1131: '@media (min-width: 1131px)',
+    max320: '@media (max-width: 320px)',
+  },
+};
```

ESLint

[error] Expected media to have a type annotation.

(@typescript-eslint/typedef)